ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -1999,6 +1999,10 @@
 			buildSettings = {
 				APPLETVOS_DEPLOYMENT_TARGET = 9.0;
 				BITCODE_GENERATION_MODE = bitcode;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
+				CLANG_WARN_COMMA = YES;
+				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
```

Comment:
Do we want these settings in the project file?

Reply:
Actually those are at project level. The four duplicated-like diffs are for each configurations (Debug, Profile, Release, Test).

Reply:
Those are automatically added by Xcode 9.4 but should be removed?

Reply:
That's what we've typically done, because we try to have them in the `.xcconfig`s instead. But I don't personally feel strongly about this.

Reply:
Once https://github.com/jspahrsummers/xcconfigs is updated this could be removed.

Reply:
Filed an issue: https://github.com/ReactiveCocoa/ReactiveCocoa/issues/3605